agent/http_register.go

```diff
@@ -69,6 +69,7 @@ func init() {
 	registerEndpoint("/v1/catalog/node/", []string{"GET"}, (*HTTPHandlers).CatalogNodeServices)
 	registerEndpoint("/v1/catalog/node-services/", []string{"GET"}, (*HTTPHandlers).CatalogNodeServiceList)
 	registerEndpoint("/v1/catalog/gateway-services/", []string{"GET"}, (*HTTPHandlers).CatalogGatewayServices)
+	registerEndpoint("/v1/catalog/exported-services/peers", []string{"GET"}, (*HTTPHandlers).PeerExportedServices)
```

Comment:
Not sure if this would be the right API path

Reply:
Wondering if we should design the API something like

```
GET /v1/catalog/exported-services?partition=<part>
GET /v1/catalog/exported-services?peer=<peer-name>
```

Both `part` and `peer-name` can also support wildcards.

The response could look something like

```json
[
  {
      "Partition": "Part-1",
      "Services": [....]
  },
  {
      "Peer": "Peer-1",
      "Services": [....]
  }
]
```

depending on the inputs. What do you think?